direct/nn/vsharp/vsharp.py

```diff
@@ -319,3 +321,264 @@ def forward(
             u = u + self.rho[admm_step] * (x - z)
 
         return out
+
+
+class LagrangeMultipliersInitializer3D(torch.nn.Module):
+    """A convolutional neural network model that initializes the Lagrange multiplier of :class:`VSharpNet3D`."""
+
+    def __init__(
+        self,
+        in_channels: int,
+        out_channels: int,
+        channels: tuple[int, ...],
+        dilations: tuple[int, ...],
+        multiscale_depth: int = 1,
+        activation: ActivationType = ActivationType.PRELU,
```

Comment:
Isn't channels and such not always length 4?

Reply:
Hmm, Not necessarily